src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_transformers.py

```diff
@@ -13,7 +13,7 @@ class TestBatchTransformers(unittest.TestCase):
 
     def test_transform_object_empty_list(self):
         fake_transformer = Transformer({"testing": "potato"})
-        result = fake_transformer.transform_result([])
+        result = fake_transformer.transform_result([]) 
```

Comment:
Let's revert this line. Just adds trailing whitespace, which I think will fail linting.

Reply:
Updated